.github/workflows/langwatch-app-ci.yml

```
@@ -0,0 +1,78 @@
+name: langwatch-app-ci
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
```

actionlint (1.7.7)

string should not be empty

(syntax-check)